src/games/gameitem.py

```diff
@@ -285,9 +290,13 @@ def update(self, message, old_client=None):
         # Spawn announcers: IF we had a gamestate change, show replay and hosting announcements
         if oldstate != self.state:
             if self.state == "playing":  # The delay is there because we have a 5 minutes delay in the livereplay server
-                QtCore.QTimer.singleShot(5*60000, self.announceReplay)
+                self.launched_at = message['launched_at']
+                if time.time() - self.launched_at > 5*60:  # most already running games at client start
+                    self.announceReplay()
+                else:
+                    QtCore.QTimer().singleShot(5*60100 - 1000*(time.time() - self.launched_at), self.announceReplay)
```

Comment:
This works for now, but we definitely need a comment explaining where that 60100 came from. Honestly all our woes from the timer come from the fact that we really need to have a single place away from any widgets that announces replay appearance.

I did some work in #709 that centralizes processing games from the server - it's some broad and destructive refactoring that can probably be merged no sooner than 0.14, but if it does, we'll be able to make a single class that reports live replays after a delay and just hook up to it.